connection.js

```diff
@@ -713,20 +712,16 @@ function get_r_u_arr_booleans(familyNameString, notificationRanked) {
 
 function getBooleans_r_u_Reject(argNameString, nameF, ending) {
     const rankedUnranked = beforeRankedUnrankedGamesSpecial("for ", "", argNameString, "");
-    const msg = `${nameF} not allowed on this bot ${rankedUnranked}${ending}.`;
-    conn_log(msg);
-    return { reject: true, msg };
+    return getReject(`${nameF} not allowed on this bot ${rankedUnranked}${ending}.`);
 }
 
 function getBoardsizeNotSquareReject(argNameString, notificationWidth, notificationHeight) {
     const rankedUnranked = beforeRankedUnrankedGamesSpecial("for ", "", argNameString, "");
-    conn_log(`boardsize ${notificationWidth}x${notificationHeight} `
-             + `is not square, not allowed ${rankedUnranked}`);
-    const msg = `Your selected board size ${notificationWidth}x${notificationHeight} `
-                + `is not square, not allowed ${rankedUnranked}, `
-                + `please choose a SQUARE board size (same width and `
-                + `height), for example try 9x9 or 19x19}`;
-    return { reject: true, msg };
+    const reason = `Selected board size ${notificationWidth}x${notificationHeight} `
+                   + `is not square, not allowed ${rankedUnranked}`;
+    const demand = ` , please choose a SQUARE board size (same width and `
+                   + `height), for example try 9x9 or 19x19}`;
+    return getReject(reason, demand);
```

Comment:
Especially here i want to see unit testing. For example, there seem to be a random extra space at the start of  `demand`.

Reply:
indeed